docs/project/ide/boxes.md

```diff
@@ -479,6 +481,37 @@ chdir /home/codio/workspace
 # Specify the process/command to start, e.g.
 exec your-service-name
 ```
+#### Ubuntu 18.04 Boxes
```

Comment:
we have https://github.com/codio/install_software/blob/master/package_list.json#L110 which allow defining scripts for autostart easier as just a bash script